source/_components/binary_sensor.august.markdown

```diff
@@ -21,3 +21,7 @@ If you have August Doorbell, once you have enabled the [August component](/compo
 * Doorbell motion sensor
 * Doorbell online sensor
 
+If you have August Smart Lock with DoorSense, once you have enabled the [August component](/components/august/), you should see following sensor:
```

Comment:
`you should see the following sensors:`

Reply:
I'm assuming here to insert the word "the"? Just did that.

If not, not sure what is meant then.